pallets/manta-pay/src/lib.rs

```diff
@@ -806,13 +808,18 @@ where
     {
         sources
             .map(move |(account_id, withdraw)| {
-                FungibleLedger::<T>::can_withdraw(asset_id.0, &account_id, withdraw.0)
-                    .map(|_| WrapPair(account_id.clone(), withdraw))
-                    .map_err(|_| InvalidSourceAccount {
-                        account_id,
-                        asset_id,
-                        withdraw,
-                    })
+                FungibleLedger::<T>::can_reduce_by_amount(
+                    asset_id.0,
+                    &account_id,
+                    withdraw.0,
```

Comment:
For me this tuple access is confusing to read especially on withdraw, but also on asset_id.
Any chance we can make these named accessors  in the future?

Reply:
This change is actually not from this PR, it's already in master and manta-rs. They probably can be named, what do you think @bhgomes

Reply:
This is because `AssetId` and `AssetValue` are wrapped types in `manta-rs`, so we just unwrap them by one level. It's a 1-tuple.